scripts/build-single-packages.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+const fse = require('fs-extra');
+const glob = require('glob');
+
+const root = process.cwd();
+const packageJson = require(`${root}/package.json`);
+const configJson = require(`${root}/${process.argv[process.argv.indexOf('--config') + 1]}`);
+
+const foldersBlackList = configJson.blackList ? configJson.blackList : []
+
+if (!configJson.modules && configJson.modules.length !== 0) {
```

Comment:
If `configJson.modules` is undefined wouldn't `configJson.modules.length` always throw an error?

Reply:
Config file check prevent this now.